packages/@okta/vuepress-site/docs/guides/add-an-external-idp/before-you-begin/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Before you begin
+---
+You want to give your users the freedom to choose which Identity Provider that they use to sign in to your application. Okta manages connections to Identity Providers for your application, sitting between your application and the Identity Provider that authenticates your users.
+
+We also support additional services such as directories and credential providers. See the [Okta Integration Network Catalog](https://www.okta.com/okta-integration-network/) to browse all integrations by use case.
+
+This guide assumes that you:
+
+* Have an Okta Developer Edition organization. (Don't have one? [Create one for free](https://developer.okta.com/signup).)
+* Have an application that you want to add authentication to.
```

Comment:
Are we also making an assumption that the developer knows which provider they would like to use? Is that worth calling out?

It seems like we're making that assumption; this guide will not explain the differences between SAML and OIDC, and will not help you choose between them.

Reply:
The first section that I had was on "Choosing a Provider", but since we don't have that UI in place yet and it isn't on the roadmap, I removed it and just left the Before you begin. I also forgot to put in a link to the new Concept piece on External Identity Providers. So, thank you!